AdobeStockAdminUi/Block/Adminhtml/System/Config/TestConnection.php

```diff
@@ -104,7 +106,7 @@ public function isConnectionSuccessful(): bool
     public function getAjaxUrl(): string
     {
         return $this->_urlBuilder->getUrl(
-            'adobe_stock/system_config/testconnection',
+            self::XML_PATH_TEST_CONNECTION,
```

Comment:
Hi @RakeshJesadiya, what is the reason for adding the constant if it's private and used only ones?